test/flake.nix

```diff
@@ -18,32 +18,30 @@
           # Required for elastic search
           config.allowUnfree = true;
         };
-        process-compose = {
-          postgres = {
-            imports = [
-              inputs.services-flake.processComposeModules.default
-              ../nix/postgres_test.nix
-            ];
-          };
-          redis = {
-            imports = [
-              inputs.services-flake.processComposeModules.default
-              ../nix/redis_test.nix
-            ];
-          };
-          redis-cluster = {
-            imports = [
-              inputs.services-flake.processComposeModules.default
-              ../nix/redis-cluster_test.nix
-            ];
-          };
-          elasticsearch = {
-            imports = [
-              inputs.services-flake.processComposeModules.default
-              ../nix/elasticsearch_test.nix
-            ];
-          };
-        };
+        process-compose =
+          let
+            mkPackageFor = mod:
+              let
+                # Derive name from filename
+                name = lib.pipe mod [
+                  builtins.baseNameOf
+                  (builtins.match "(.*)_test.nix")
```

Comment:
Be careful with `builtins.match` regex parsing. IIRC it has issues on darwin, but not on Linux (going by what @ursi once told me).

Reply:
I think it's fine if we have CI for both platforms already.